app/models/concerns/searchable.rb

```diff
@@ -5,15 +5,14 @@ module Searchable
     include PgSearch
     include SearchCache
 
-    pg_search_scope :pg_search, {
-      against: :ignored, # not used since using a tsvector_column
-      using: {
-        tsearch: { tsvector_column: 'tsv', dictionary: "spanish", prefix: true }
-      },
-      ignoring: :accents,
-      ranked_by: '(:tsearch)',
-      order_within_rank: (self.column_names.include?('cached_votes_up') ? "#{self.table_name}.cached_votes_up DESC" : nil)
-    }
+    pg_search_scope :pg_search,
```

Comment:
I find searcheable.rb clearer as it was (both indentation and curly braces help readability here I think)